docker-build/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Dockerfile for bitnami/fluentd sidecar image with all the necessary plugins for our log analytic providers
+FROM bitnami/fluentd:latest
```

Comment:
we should _never_ use `latest`. This causes issues when `bitnami` makes some breaking change and it breaks our stuff. This will lead to lots of support calls.